pyramid_oereb/standard/xtf_import/geometry.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
+import logging
+
+from geoalchemy2.shape import from_shape
+from pyreproj import Reprojector
+from shapely.geometry import MultiPoint, MultiLineString, MultiPolygon, GeometryCollection, Point, \
+    LineString, Polygon
+
+from pyramid_oereb.standard.xtf_import.util import parse_string, parse_ref, get_tag, stroke_arc
+
+
+class Geometry(object):
+
+    TAG_POINT_LV03 = 'Punkt_LV03'
+    TAG_POINT_LV95 = 'Punkt_LV95'
+    TAG_LINE_LV03 = 'Linie_LV03'
+    TAG_LINE_LV95 = 'Linie_LV95'
+    TAG_AREA_LV03 = 'Flaeche_LV03'
+    TAG_AREA_LV95 = 'Flaeche_LV95'
+    TAG_LAW_STATUS = 'Rechtsstatus'
+    TAG_PUBLISHED_FROM = 'publiziertAb'
+    TAG_GEO_METADATA = 'MetadatenGeobasisdaten'
+    TAG_PUBLIC_LAW_RESTRICTION = 'Eigentumsbeschraenkung'
+    TAG_RESPONSIBLE_OFFICE = 'ZustaendigeStelle'
+    TAG_COORD = 'COORD'
+    TAG_ARC = 'ARC'
+
+    def __init__(self, session, model, geometry_type, arc_max_diff=0.001, arc_precision=3):
+        self._session = session
+        self._model = model
+        self._geometry_type = geometry_type
+        self._arc_max_diff = arc_max_diff
+        self._arc_precision = arc_precision
+        self._reproject = Reprojector().get_transformation_function(from_srs=21781, to_srs=2056)
```

Comment:
With the model is the reprojection function still used?

Reply:
Good point! I'll read the target projection from the configuration file.